ast/compile_test.go

```diff
@@ -2976,7 +2976,7 @@ func TestRewriteDeclaredVars(t *testing.T) {
 				p = true {
 					__local0__ = 10
 					__local3__ = [1]
-					every __local1__, __local2__ in __local3__ { __local2__ == 1 }
+					every __local1__, __local2__ in __local3__ { __local2__ = 1 }
```

Comment:
I was confused by the lone `__local1__`, but when I saw in the trace that it's used for lookup in `__local3__`, it makes sense. I think .. This is stuff I haven't looked at until now 😅 .